_includes/layout.liquid

```diff
@@ -2,9 +2,20 @@
 <html lang='en'>
 
 <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id={{google-id}}"></script>
+    <script>
```

Comment:
⏳ Should we not load this if the optional google tracking id is not provided? :)

Reply:
Fixed.